src/styles/summernote/common.scss

```
@@ -63,6 +63,90 @@ $img-margin-right: 10px;
+      }
+
+      ul > li::before          { content: "•"; }
+      ul ul > li::before       { content: "◦"; }
+      ul ul ul > li::before    { content: "▪"; }
+
+    }
+  }
+}
+
+// Fix Firefox cursor position in empty list items
+@-moz-document url-prefix() {
```

Stylelint (17.3.0)

[error] Unexpected vendor-prefixed at-rule "@-moz-document" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)